.env.example

```
@@ -1,28 +1,86 @@
+# OAUTH CONFIGURATION - Create new apps and replace these
+# =============================================================================
+GOOGLE_CLIENT_ID="your_google_client_id_here"
+GOOGLE_CLIENT_SECRET="your_google_client_secret_here"
+FACEBOOK_APP_ID="your_facebook_app_id_here"
+FACEBOOK_APP_SECRET="your_facebook_app_secret_here"
+
+# =============================================================================
+# JWT CONFIGURATION - Generate secure secrets
+# =============================================================================
+# Generate with: node -e "console.log(require('crypto').randomBytes(64).toString('hex'))"
+JWT_SECRET="your_jwt_secret_here_64_chars_minimum"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)